### DISCUSSION TITLE
Support with Shop

### DISCUSSION THREAD

**Discussion by NotEvenAUnicorn** [GriefPrevention/GriefPrevention#1464] [Q&A]
Implement chest protection with [Shop](https://www.spigotmc.org/resources/shop-a-simple-intuitive-shop-plugin.9628/). Players can open doublechests with shop signs on them if they are trusted on the land the shop is in, and they are not supposed to be able to. The shop plugin has a cancel inventory event which is ignored by GP.

**Comment by Jikoo** [comment]
GP does not un-cancel events nor forcibly open containers. You'll need to contact the dev of that plugin - it's closed source, we can't help track down conflicts.

**Reply by NotEvenAUnicorn** [reply]
I spoke with Shop's dev first, he told me to come here.

**Reply by Jikoo** [reply]
Again, GP does not un-cancel events and does not forcibly open containers. Without further information, it's absolutely not GP at fault. If they can provide any useful information that's great, but since it's closed source we cannot help you track the issue down further.

**Comment by RoboMWM** [comment]
Yea, GP doesn't uncancel events.

A suggestion is to change the design with PrettySimpleShop and having GP be responsible for protecting the shop instead of the shop plugin itself. Also, it's free.
